Common/itkParabolicErodeDilateImageFilter.h

```diff
@@ -142,8 +142,8 @@ class ITK_TEMPLATE_EXPORT ParabolicErodeDilateImageFilter : public ImageToImageF
   void
   GenerateData() override;
 
-  int
-  SplitRequestedRegion(int i, int num, OutputImageRegionType & splitRegion);
+  unsigned int
+  SplitRequestedRegion(unsigned int i, unsigned int num, OutputImageRegionType & splitRegion) override;
```

Comment:
@mstaring @stefanklein I think this change is really necessary, otherwise the special `SplitRequestedRegion` version of ParabolicErodeDilateImageFilter will not be called!  `ParabolicErodeDilateImageFilter::SplitRequestedRegion` is special, because it avoids the "current dimension", when it estimates the `splitAxis`.

This bug must have been introduced with an upgrade of ITK after June 14, 2011. I wonder how it could remain unnoticed for so long! Because ParabolicErodeDilateImageFilter is actually being used by elastix!

Reply:
It appears that our ParabolicErodeDilateImageFilter is not tested at the CI, except for being used indirectly by the `ElastixLib.SingleSliceMaskedTranslation3D` GoogleTest unit test (until now). But it still appears obvious that its SplitRequestedRegion member function was intended to override the corresponding member function of ITK's ImageSource. So I'll just merge this now.